README.md

```diff
@@ -132,7 +132,7 @@ Storybook will be available at [http://localhost:26638/](http://localhost:26638/
 pnpm run docs
 ```
 
-The docs will be available at [http://localhost:3000/](http://localhost:3000/)
+The docs will be available at [http://localhost:3001/](http://localhost:3001/)
```

Comment:
In #2179 the second site runs with 3001, could we use something else 😅

Reply:
Changed to `3300` 😄